frontend/package.json

```diff
@@ -12,7 +12,7 @@
     "@react-hook/resize-observer": "^1.2.6",
     "@silevis/reactgrid": "3.1",
     "@trussworks/react-uswds": "4.1.1",
-    "@ttahub/common": "^2.2.1",
+    "@ttahub/common": "^2.1.12",
```

Comment:
this is going to conflict with the DEI changes. You have to merge main and republish your changes

Reply:
yea i increased it to 2.2.2.